lib/diplomat/health.rb

```diff
@@ -41,10 +41,12 @@ def service(s, options = {})
       custom_params << ['passing'] if options[:passing]
       custom_params << use_named_parameter('tag', options[:tag]) if options[:tag]
       custom_params << use_named_parameter('near', options[:near]) if options[:near]
+      custom_params << use_named_parameter('node-meta', options[:node_meta]) if options[:node_meta]
```

Comment:
is it node-meta or node_meta?

Reply:
consul calls it node-meta. But since ruby does not love hyphens in param names I used the snake case convention and called it node_meta. Do you think it should be different?